dissect/target/loaders/log.py

```diff
@@ -19,10 +30,11 @@ def map(self, target: Target) -> None:
         self.target = target
         vfs = VirtualFilesystem(case_sensitive=False)
         for entry in self.path.parent.glob(self.path.name):
-            ext = entry.suffix.lower()
+            ext = self.options.get("hint", entry.suffix.lower()).strip(".")
             if (mapping := self.LOGS_DIRS.get(ext, None)) is None:
                 continue
             mapping = str(Path(mapping).joinpath(entry.name))
             vfs.map_file(mapping, str(entry))
         target.filesystems.add(vfs)
         target.fs = vfs
+        target.datetime = datetime.today()
```

Comment:
What’s this for?

Reply:
The plugin expects time zone information but because this is not a regular target, no such thing is available, this is just to make sure the plugin does not crash.

Reply:
But you are right, this is not a very clean solution, I will improve it.